integration-test/background/click-to-load-facebook.js

```diff
@@ -1,23 +1,67 @@
 const harness = require('../helpers/harness')
+const { logPageRequests } = require('../helpers/requests')
 
 const testSite = 'https://privacy-test-pages.glitch.me/privacy-protections/click-to-load/'
+const facebookDomains = new Set(['facebook.com', 'facebook.net'])
```

Comment:
I'd also consider fbcdn.net (https://slayterdev.github.io/tracker-radar-wiki/entities/Facebook,%20Inc..html) it loads quite a bit of stuff on the privacy test page.

Reply:
Gotya, Done.